src/Message/CalculateSalesTaxRequest.php

```diff
@@ -84,6 +84,13 @@ public function getData()
             throw new InvalidRequestException('Either the amount or items parameter is required.');
         }
 
+        if ($this->getCustomerId() === null && $this->getCustomerReference() === null) {
+            // With the Avalara tax engine, a customer ID is required for tax to be calculated
+            // Vindicia's advice is to pass a dummy ID if you just want to calculate tax
+            // without a specific user
+            $this->setCustomerId('Dummy ID for tax calculation');
```

Comment:
Should we have some unique ID and make it a constant instead of passing a string?

for example 
```const DUMMY_CUSTOMER_FOR_TAX_CALCULATION = dummy123```

Reply:
I think the name should be something ridiculously long so it is unlikely that it ever clashes with a real user for anything. I'm fine with it being a const or not. I didn't bother making one since it only needs to be used in one place and I thought it might be easier to read if it was all kept in one place, but if other people prefer a const too, I'll change it!

Reply:
I agree with Parth, I think making it a constant could make it easier for debugging later. It also gives a specific ID to look for instead of searching for a string. I just think it's better practice for the future.